File: Programming Basics/02-Conditional Statements - Exercise/lunchBreak.js
function lunchBreak(input) {

    let tvShow = input[0];
    let episodeDuration = Number(input[1]);
    let breakDuration = Number(input[2]);

    let lunchTime = breakDuration * (1/8);
    let restTime = breakDuration * (1/4);

    let freeTimetoWatchTvShow = breakDuration - (lunchTime + restTime);
    let totalTime = Math.abs(freeTimetoWatchTvShow - episodeDuration);

    if(freeTimetoWatchTvShow >= episodeDuration) {
        console.log(`You have enough time to watch ${tvShow} and left with ${Math.ceil(totalTime)} minutes free time.`);
    } else {
        console.log(`You don't have enough time to watch ${tvShow}, you need ${Math.ceil(totalTime)} more minutes.`);
    }
    
}

lunchBreak(["Teen Wolf",
"48",
"60"])

File: Programming Basics Java/src/ConditionalStatementsExercise/LunchBreak.java
package ConditionalStatementsExercise;

import java.util.Scanner;

public class LunchBreak {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String movieTitle = scanner.nextLine();
        int durationOfMovie = Integer.parseInt(scanner.nextLine());
        int durationOfBreak = Integer.parseInt(scanner.nextLine());

        double timeForLunch = durationOfBreak * 0.125;
        double timeForRest = durationOfBreak * 0.25;
        double timeForMovie = durationOfBreak - (timeForRest + timeForLunch);

        double difference = Math.ceil(Math.abs(timeForMovie - durationOfMovie));

        if (timeForMovie >= durationOfMovie) {
            System.out.printf("You have enough time to watch %s and left with %.0f minutes free time.", movieTitle, difference);
        } else {
            System.out.printf("You don't have enough time to watch %s, you need %.0f more minutes.", movieTitle, difference);
        }
    }
}
